src/components/Welcome/index.tsx

```diff
@@ -79,7 +81,11 @@ const Welcome: FunctionComponent<StepComponentBaseProps> = ({
 
   return (
     <ScreenLayout actions={actions} className={style.container}>
-      <PageTitle title={welcomeTitle} subTitle={welcomeSubTitle} />
+      <PageTitle
+        title={welcomeTitle}
+        subTitle={welcomeSubTitle}
+        shouldAutoFocus={isFirstScreen && autoFocusOnInitialScreenTitle}
```

Comment:
I see this prop being passed in several places, but I don't see a change to `PageTitle` itself to do anything with the prop.
Did you forget a commit, or am I missing something?

Reply:
This prop already exists in [`PageTitle`](https://github.com/onfido/onfido-sdk-ui/blob/master/src/components/PageTitle/index.tsx#L23) as it was introduced in an earlier PR #1560 hence why there isn't any change to `PageTitle` in this PR.

Reply:
I see, I was looking at the wrong branch, my mistake